de-lib/src/main/java/org/iplantc/de/client/services/impl/AppUserServiceFacadeImpl.java

```diff
@@ -230,25 +230,19 @@ protected Void convertFrom(String object) {
         });
     }
 
+
     @Override
-    public void getAppDetails(final App app, DECallback<App> callback) {
-        String address = APPS + "/" + app.getSystemId() + "/" + app.getId() + "/details";
+    public void getAppDetails(final String appId,
+                              String systemId,
+                              DECallback<Splittable> callback) {
+        String address = APPS + "/" + systemId + "/" + appId + "/details";
 
         ServiceCallWrapper wrapper = new ServiceCallWrapper(GET, address);
 
-        deServiceFacade.getServiceData(wrapper, new DECallbackConverter<String, App>(callback) {
-            @Override
-            protected App convertFrom(String object) {
-                Splittable split = StringQuoter.split(object);
-                AutoBean<App> appAutoBean = AutoBeanUtils.getAutoBean(app);
-                AutoBeanCodex.decodeInto(split, appAutoBean);
-                return appAutoBean.as();
-            }
-        });
```

Comment:
As @ash3rz figured, this is the place where we trying to mutate the selected `app`. Removed this code and modified all it return a splittable.